pkgs/development/libraries/libusbp/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, udev
+, cmake
+, pkg-config
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
I'd recommend `stdenv.mkDerivation(finalAttrs: {`.

Reply:
All done:
- using pkgs/by-name
- added the stepper motor package "pololu-tic" as a separate commit
- using finalAttrs